src/Core/Models/Business/TaxInfo.cs

```diff
@@ -37,12 +37,24 @@ public string TaxIdType
 
             switch (BillingAddressCountry)
```

Comment:
It looks Good. 

If this switch grows or changes frequently, you might consider refactoring it. Using a Dictionary<string, string> for mapping country codes to tax ID types can be more efficient and easier to maintain.

```
private static readonly Dictionary<string, string> TaxIdTypeMap = new Dictionary<string, string>
{
    { "AD", "ad_nrt" },
    { "AE", "ae_trn" },
    { "AF", "af_tin" }, 
    { "AR", "ar_cuit" },
    // ... add all other mappings ...
    { "ZA", "za_vat" },
};
```

Reply:
Nice, thanks for these suggestions. I'll push up a change to address them